docs/docs/plugins.md

```diff
@@ -2,42 +2,50 @@
 title: Plugins
 ---
 
-Plugins are Node.js packages that implement Gatsby APIs. They enable you to
-easily solve common website build problems e.g. setup Sass, add markdown
-support, process images, etc.
+One of the best ways to add functionality to Gatsby is through our plugin system. Gatsby is designed to be extensible, which means plugins are able to extend and modify just about everything Gatsby does.
 
-For larger / complex sites, they let you modularize your site customizations
-into site-specific plugins.
+Of the many possibilities, plugins can:
 
-Gatsby has a large and growing set of plugins. To search/browse official and
-community plugins and their documentation, visit the [Plugin Library](/plugins/).
+- add external data or content (e.g. your CMS, static files, a REST API) to your Gatsby GraphQL data
+- transform data from other formats (e.g. Markdown, YAML, CSV) to JSON objects
+- add third-party services (e.g. Google Analytics, Instagram) to your site
+- anything you can dream up!
 
-For documentation on the different types of plugins and the functionality provided by each, see the [Plugin Authoring page](/docs/plugin-authoring/).
+Gatsby plugins are Node.js packages that implement Gatsby APIs. For larger, more complex sites, plugins let you modularize your site customizations into site-specific plugins.
 
-For a walkthrough of how to build and publish your own plugin, see the [Source Plugin Tutorial](/docs/source-plugin-tutorial/).
+## Search published plugins
 
-## How to use Gatsby plugins?
+Gatsby has a large and growing ecosystem of official and community plugins. To browse plugins and their documentation, visit the [Gatsby Plugin Library](/plugins/).
 
-Gatsby plugins are Node.js packages, so you can install them like other packages in
-node using NPM.
+## Learn more about plugins
 
-For example, `gatsby-transformer-json` is a package which adds support for JSON
-files to the Gatsby data layer.
+For documentation with further detail on what comprises a Gatsby plugin (file structure, etc), see the [plugin authoring page](/docs/plugin-authoring/).
+
+## Build and publish a plugin
+
+For a walkthrough of how to build and publish your own plugin, see the [source plugin tutorial](/docs/source-plugin-tutorial/).
```

Comment:
would it make sense to link to a stub article(s) for other plugin tutorials (transformer plugin tutorial, for example)?

Reply:
Yes! Great idea. Let me know if I'm missing any:

- [source plugin tutorial](https://next.gatsbyjs.org/docs/source-plugin-tutorial/)
- [transformer plugin tutorial](https://next.gatsbyjs.org/tutorial/part-six/#part-six) (aka tutorial part six)
- [source plugin example usage](https://next.gatsbyjs.org/tutorial/part-five/#source-plugins) (aka tutorial part five)